lib/config.php

```diff
@@ -151,6 +151,9 @@
 		'default_mailserver_ip' => '127.0.0.1',
 		'default_mx' => 'localhost'
 	),
+	'voip' => array(
+		'default_call_recording' = false
```

Comment:
Dlaczego default_call_recording jest true lub false skoro flaga nagrywania to mapa bitowa (obecnie nagrywanie na zlecenie operatora + nagrywanie na zlecenie klienta)?